var/spack/repos/builtin/packages/jq/package.py

```diff
@@ -34,3 +34,4 @@ class Jq(AutotoolsPackage):
     version('1.5', 'c8070bd6ec275404f77db3d2e568c9a3')
 
     depends_on('oniguruma')
+    depends_on('bison@3.0:')
```

Comment:
It was almost certainly picking up the copy of `bison` in my linuxbrew directory.

I have a circular headache, my mirror of the spack tarballs lives in git-lfs.  The git bits in the base system aren't new enough to be able to clone/checkout them.  I used linuxbrew to build those bits but I have to be careful to get them out of the way again before I let loose the spack build.

Isolation is a PITA.  Along those lines, is there a way to get `spack` to ignore `~/.spack`?  I keep tripping over things that show up there (e.g. compilers.yaml).